app/src/main/java/com/kkolontay/theworld/viewmodel/CountryViewModel.kt

```diff
@@ -0,0 +1,54 @@
+package com.kkolontay.theworld.viewmodel
+
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.kkolontay.theworld.flow.Flows
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+class CountryViewModel : ViewModel() {
+   private val _timerState: MutableStateFlow<Int> = MutableStateFlow(0)
+    val timerState = _timerState.asStateFlow()
+    private val _tap: MutableStateFlow<Int> = MutableStateFlow(0)
+    val tap = _tap.asStateFlow()
+    private val _back: MutableStateFlow<Int> = MutableStateFlow(0)
+    val back = _back.asStateFlow()
+    val flows = Flows
+
+    init {
+
+        viewModelScope.launch {
```

Comment:
I don't believe you need three separate calls to viewModelScope, here. You should be able to put all the flow collection calls inside a singe viewModelScope block, it should do exactly the same thing but would be more concise and readable.

Reply:
will delete that, because don't need that viewmodel any more.